README.md

```diff
@@ -929,6 +930,79 @@ config.serializationConfig.portableFactories[1] = new PortableFactory();
 
 Note that the ID that is passed to the `SerializationConfig` is same as the `factoryId` that `Customer` object returns.
 
+
+### 4.2.1. Versioning for Portable Serialization
+
+More than one version of the same class may need to be serialized and deserialized. For example, a client may have an older version of a class and the member to which it is connected may have a newer version of the same class.
+
+Portable serialization supports versioning. It is a global versioning, meaning that all portable classes that are serialized through a member get the globally configured portable version.
+
+You can declare the version in the `hazelcast.xml` configuration file using the `portable-version` element, as shown below.
+	
+```xml
+<hazelcast>
+    ...
+	<serialization>
+	    <portable-version>1</portable-version>
+	</serialization>
+	...
+</hazelcast>
+```
+
+If you update the class by changing the type of one of the fields or by adding a new field, it is a good idea to upgrade the version of the class, rather than sticking to the global version specified in the `hazelcast.xml` file.
```

Comment:
```suggestion
If you update the class by changing the type of one of the fields or by adding a new field, it is a good idea to upgrade the version of the class, rather than sticking to the global version specified in the `hazelcast-client.json` file.
```